MapCore/src/main/java/org/wwarn/mapcore/client/components/customwidgets/facet/FacetSearchableCheckBoxWidget.java

```diff
@@ -57,11 +62,12 @@
     public static final String STYLE_CHECKBOXLIST_ITEM_CHECKED = "searchCheckBoxListItemChecked";
     public static final String STYLE_CHECKBOXLIST_ITEM_DISABLED = "searchCheckBoxListItemDisabled";
     public static final String STYLE_INITIAL_SEARCHBOX_TEXT = "initialSearchBoxText";
-    public static final String STYLE_SELECTION_CLEAR_ANCHOR = "searchableCheckBoxClearAnchor";
```

Comment:
If you remove this bit, you need to be sure that any implementation is using this class in the css file